environment.yml

```diff
@@ -69,6 +69,10 @@ dependencies:
   - typing >=3.6.6
   # Pin OpenSSL to avoid: https://github.com/DIRACGrid/DIRAC/issues/4489
   - openssl <1.1
+  # OAuth2
```

Comment:
These are needed in `setup.cfg` as well.

Reply:
added